deb-get

```diff
@@ -532,6 +532,19 @@ function deb_quickemu() {
     SUMMARY="Quickly create and run optimised Windows, macOS and Linux desktop virtual machines."
 }
 
+function deb_virtualbox() {
+    ARCHS_SUPPORTED="amd64"
+    if [ ! -e  /usr/share/keyrings/oracle-virtualbox-2016.gpg ] ; then
```

Comment:
This should only run when installing, and a counterpart should be provided when removing. Ideally, this should not be required at all.

Reply:
Without it I was getting failures, it only runs when the key isn't there and the inbuilt  removal code already removes it I think (or the deb uninstall maybe).
Will look further as I may have used this to fix a typo I also already fixed or some such,,,,, it is very late(early) 

[![Tests](https://github.com/philclifford/deb-get/actions/workflows/tests.yml/badge.svg?branch=test-virtualbox)](https://github.com/philclifford/deb-get/actions/workflows/tests.yml)

[here is the run](https://github.com/philclifford/deb-get/runs/7532056726?check_suite_focus=true#step:5:16)

Reply:
```
Err:15 https://download.virtualbox.org/virtualbox/debian jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A2F683C52980AECF
Fetched 154 kB in 5s (31.8 kB/s)
Reading package lists...
[...]
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://download.virtualbox.org/virtualbox/debian jammy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A2F683C52980AECF
W: Failed to fetch https://download.virtualbox.org/virtualbox/debian/dists/jammy/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A2F683C52980AECF
W: Some index files failed to download. They have been ignored, or old ones used instead.
Selecting previously unselected package virtualbox.
```
which goes on to  install on my system because I have "old ones" and tonight (ahem - this 3am actually) ...
So I think best to leave it there and sleep on it
Cheers

Reply:
So,  now I remember what a pain Virtualbox upstream could be, and why I generally avoided the "debian" packaging.
Trying to rename the app to `virtualbox-6.1` so you actually get that not whatever ubuntu is providing in `virtualbox` ...
well now I broke it so it is marked draft while I go quietly mad(der)

Reply:
But is this the actual name of the package? Because if not, it would break the detection of the package being installed.

Reply:
Yes, they have `virtualbox-6.0` and `virtualbox-5.2` in the same repo too.
Making the list name just `virtualbox` might be the lesser of two evils,
but as this stands it now works and actually installs (and recognises) the
upstream version rather than picking whatever is in the distro repository.